parser_library/test/processing/mach_instr_test.cpp

```diff
@@ -232,3 +232,9 @@ SYM  DS   CL1
     a.collect_diags();
     ASSERT_EQ(a.diags().size(), 0U);
 }
+
+TEST(mach_instr_processing, verify_rel_addr_size)
+{
+    for (const auto& instr : context::instruction::machine_instructions)
```

Comment:
are we `EXPECT`ing anything here? or just relying on the assert in `generate_reladdr_bitmask`?